cli_tools/gce_ovf_import/daisy_utils/data_disk_workflow_updater.go

```diff
@@ -17,16 +17,37 @@ package daisyovfutils
 import (
 	"fmt"
 
-	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_ovf_import/ovf_utils"
-	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
 	"google.golang.org/api/compute/v1"
+
+	ovfutils "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_ovf_import/ovf_utils"
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
 )
 
 const (
+	cleanupStepName        = "cleanup"
 	createInstanceStepName = "create-instance"
 	gceMinimumDiskSizeGB   = "10"
 )
 
+// AddDataDisksToInstanceImport updates the CreateInstances step in w to include
+// disks based on the referenced images. The cleanup step is updated to delete the
+// images.
```

Comment:
This is a good comment -- since it's doing two things, I removed the cleanup portion, since it's just a one-liner, and it makes for cleaner names and tests.